### PR TITLE
brew science depracated

### PR DESCRIPTION
**Pull request by Arshiamidos**

### CODE REVIEW

**Comment by paulinus** [comment]
Do you know if

**Comment by paulinus** [comment]
Good to know, thanks!
Do you know on which repo do `opencv` and `ceres-solver` formulas live?
Is it in `brewsci/bio` and `brewsci/science`?
I'm wondering if those are needed

**Comment by Arshiamidos** [comment]
sure @paulinus , refer to [link](https://github.com/Homebrew/homebrew-science/issues/6365)

File: doc/source/building.rst [hunk 1]
```diff
@@ -48,7 +48,8 @@ Installing dependencies on MacOSX
 
 Install OpenCV and the Ceres solver using::
 
-    brew tap homebrew/science
+    brew tap brewsci/bio
```
Comment (paulinus) [inline doc/source/building.rst]:
```suggestion
```

File: doc/source/building.rst [hunk 2]
```diff
@@ -48,7 +48,8 @@ Installing dependencies on MacOSX
...
+    brew tap brewsci/science
```
Comment (paulinus) [inline doc/source/building.rst]:
```suggestion
```

File: doc/source/building.rst [hunk 3]
```diff
@@ -48,7 +48,7 @@ Installing dependencies on MacOSX
 
 Install OpenCV and the Ceres solver using::
 
-    brew tap homebrew/science
+    brew tap brewsci/science
```
Comment (paulinus) [inline doc/source/building.rst]:
```suggestion
```
we can also skip this one

**Pull request merged**